src/workerd/jsg/value-test.c++

```diff
@@ -1304,5 +1307,33 @@ KJ_TEST("jsg::Name works") {
   e.expectEval("forSymbolShared('foo') !== forSymbolApi('foo')", "boolean", "true");
 }
 
+// ========================================================================================
+
+struct RustContext: public ContextGlobalObject {
+  kj::String getRustString(::rust::String value) {
+    return kj::str(value);
+  }
```

Comment:
Let's expand the tests to cover use of the `::rust` types in:

* [ ] JSG_STRUCT
* [ ] kj::OneOf
* [ ] kj::Maybe/jsg::Optional

While these should all Just Work(tm), let's be overly cautious and verify.